tests/ztest/zexpect/CMakeLists.txt

```diff
@@ -10,12 +10,5 @@ else()
   find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
   project(base)
 
-  if(CONFIG_CPLUSPLUS)
-    message(STATUS "adding main.cpp")
-    target_sources(app PRIVATE src/main.cpp)
```

Comment:
```
. Beside that, building with cpp is not really necessary so just drop the test.
```

Plenty of folks write tests in Cpp, it's a good idea to test that case.

Reply:
okay, i updated to keep cpp test